lib/internal/abort_controller.js

```diff
@@ -84,6 +84,17 @@ function lazyMessageChannel() {
 }
 
 const clearTimeoutRegistry = new SafeFinalizationRegistry(clearTimeout);
+const finalizers = new SafeFinalizationRegistry((signalWeakRef) => {
```

Comment:
Maybe this could have a more descriptive name, like `dependantSignalsCleanupRegistry` or similar.